src/store/engine.ts

```diff
@@ -160,37 +159,65 @@ export const engineStore = createPartialStore<EngineStoreTypes>({
     ),
   },
 
-  RESTART_ENGINE_ALL: {
-    async action({ state, dispatch }) {
-      // NOTE: 暫定実装、すべてのエンジンの再起動に成功した場合に、成功とみなす
-      let allSuccess = true;
-      const engineIds = state.engineIds;
+  RESTART_ENGINES: {
+    async action({ state, dispatch, commit }, { engineIds }) {
```

Comment:
state未使用がwarningになってそうです！
修正suggestして取り込んでみます。更に変更する際は一度pullして頂ければ！
```suggestion
    async action({ dispatch, commit }, { engineIds }) {
```